docs/advanced-ping/pingask.md

```diff
@@ -0,0 +1,66 @@
+# Ping Ask
+
+Ping now offers a new service for your content, Ping Ask!
+
+Ping Ask is a way for your viewers to submit questions and for you to be able to neatly display them on stream, perfect for AMAs, QnAs, and much more!
+
+And in true Ping fashion, its easy and quick to Setup
+
+## Setup
+
+First, we'll head on over the ask.ping.gg and log in
+
+![PingAsk Login page](https://i.imgur.com/b3e1KEC.png)
+
+Authenticate with your log in of choice and you'll be brought to the main screen, it is here that we will grab our browser source link

```

Comment:
"your login of choice" is a little funny since there is only one login method right now 😅

Reply:
talked in discord about this one